cpp/ql/test/library-tests/dataflow/external-models/validatemodels.expected

```diff
@@ -0,0 +1,4 @@
+| Dubious namespace "boost::asio" in sink model. |
+| Dubious namespace "boost::asio" in source model. |
+| Dubious namespace "boost::asio" in summary model. |
```

Comment:
I think we also need to add `:` to the regex for checking namespaces, right?

Reply:
True. "need" was a strong word, I guess 😄 I do think nested namespaces will be a fairly common thing, and each of those will be flagged as "dubious". So we should probably fix this. If you'd rather delay this as future work I'm also okay with this, though

Reply:
Done (turns out this logic isn't shared, so none of the politics I was worried about are relevant).